.github/workflows/test-suite.yml

```diff
@@ -32,29 +35,16 @@ jobs:
             zsh -n "$file"
             echo "Success! 🎉"
           done
-      - name: Install Test - No Branch - Master
+      - name: Install Test - branch "${{ matrix.branch }}"
         shell: zsh {0}
         run: |
-          # Ensure Zap not already installed
-          rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap"
           # Init
-          zsh ${{ github.workspace }}/install.zsh
+          zsh ${{ github.workspace }}/install.zsh --branch ${{ matrix.branch }}
           source "${XDG_DATA_HOME:-$HOME/.local/share}/zap/zap.zsh"
           # Test Environment Variable
           [[ -d "$ZAP_DIR" ]]
-          # Test Plugin Installation
-          plug "wintermi/zsh-lsd"
-          git -C "${ZAP_DIR}/plugins/zsh-lsd/" status || exit 1
-      - name: Install Test - Branch "release-v1"
+      - name: Test Plugin Installation
         shell: zsh {0}
         run: |
-          # Ensure Zap not already installed
-          rm -rf "${XDG_DATA_HOME:-$HOME/.local/share}/zap"
-          # Init
-          zsh ${{ github.workspace }}/install.zsh --branch release-v1
-          source "${XDG_DATA_HOME:-$HOME/.local/share}/zap/zap.zsh"
-          # Test Environment Variable
-          [[ -d "$ZAP_DIR" ]]
-          # Test Plugin Installation
           plug "wintermi/zsh-lsd"
           git -C "${ZAP_DIR}/plugins/zsh-lsd/" status || exit 1
```

Comment:
This plugin that is tested here shows an error since a binary `lsd` isn't installed, so I suggest to change it to another one that doesn't print any error messages

Reply:
We could use
```zsh
plug "zsh-users/zsh-syntax-highlighting"
git -C "${ZAP_DIR}/plugins/zsh-syntax-highlighting" status || exit 1
```

Reply:
Updated

Reply:
We need to use one that is not already being loaded, and the ERROR message means the plugin was sourced.